src/lib/bootloader/grub2base.rb

```diff
@@ -148,6 +150,27 @@ def disable_serial_console
 
   private
 
+    def system_locale
+      begin
+        language = ::Bootloader::Language.new
+        language.load
+      rescue Errno::ENOENT
```

Comment:
We don't have to change CFA now, but I'd like this PR to quote the future documentation or the ideal code usage